corehq/apps/reports/templates/reports/tableau_template.html

```diff
@@ -2,18 +2,29 @@
 {% load compress %}
 {% load hq_shared_tags %}
 {% load i18n %}
+{% load static %}
 
 {% block page_content %}
 
 {% initial_page_data "server_type" server_type %}
 {% initial_page_data "ticket" ticket %}
 {% initial_page_data "validate_hostname" validate_hostname %}
+{% initial_page_data "server_address" server_address %}
+{% initial_page_data "target_site" target_site %}
+{% initial_page_data "domain_username" domain_username %}
 {% initial_page_data "view_url" view_url %}
+{% registerurl 'tableau_visualisation_ajax' request.domain %}
 
 <script src="https://{{ validate_hostname }}/javascripts/api/tableau-2.5.0.min.js"></script>
 <script src="{% static 'reports/js/tableau.js' %}"></script>
 
 <body>
+    <div id="loadingDiv">
+        <i class='fa fa-spin fa-spinner' style="font-size: 70px; margin: 50px"></i>
```

Comment:
Inline styles get hard to maintain over time. I looked at the big spinners in the HQ dashboard tiles and found this `fa-5x` class, would using that get you a similar effect? https://github.com/dimagi/commcare-hq/blob/165b91e968bc9e37f9ebb26128ff7c0d2abc10b6/corehq/apps/dashboard/templates/dashboard/base.html#L55